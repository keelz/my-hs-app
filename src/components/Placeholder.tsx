import * as React from 'react';

interface PlaceholderProps {
  text: string;
}

const Placeholder: React.SFC<PlaceholderProps> = props =>
  <div>{props.text}</div>;

export default Placeholder;
