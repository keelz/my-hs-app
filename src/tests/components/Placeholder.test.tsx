import * as React from 'react';
import * as enzyme from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Placeholder from '../../components/Placeholder';

describe('Placeholder', () => {
  describe('snapshots', () => {
    it('renders correctly', () => {
      const tree = enzyme.shallow(<Placeholder text="test" />);
      expect(shallowToJson(tree)).toMatchSnapshot();
    });
  });
});
