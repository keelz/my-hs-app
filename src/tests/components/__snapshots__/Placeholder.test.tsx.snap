// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Placeholder snapshots renders correctly 1`] = `
<div>
  test
</div>
`;
